docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
```

Comment:
Apologies, this lacks some information. It should be something along the lines of "If more Jenkins plugins are required, they can be configured by providing a `jenkins_plugins` variable to Ansible. The variable is a list with the format:

```
 -
    name: "managed-scripts"
    version: 1.3
    archive: "https://updates.jenkins-ci.org/download/plugins/managed-scripts/1.3/managed-scripts.hpi"
```
"